addon/components/freestyle-menu/index.hbs

```diff
@@ -11,15 +11,17 @@
       <LinkTo @query={{hash f=null s=section.name ss=null}} class="FreestyleMenu-itemLink">
         {{section.name}}
       </LinkTo>
-      {{#each section.subsections as |subsection|}}
+      {{#if section.subsections.length}}
         <ul class="FreestyleMenu-submenu">
```

Comment:
The `ul` tag was placed outside of the each loop. Also added an if check to avoid rendering empty `ul` tags in case of zero subsections.